torch/_C/_distributed_c10d.pyi

```diff
@@ -407,9 +407,7 @@ class ProcessGroupNCCL(ProcessGroup):
         size: int,
         timeout: timedelta,
     ): ...
-    @staticmethod
     def _group_start() -> None: ...
-    @staticmethod
```

Comment:
Removing static method as `_group_end()` might need to check the communicator map of the `ProcessGroup` to properly wait on collectives if nonblocking is used.